packages/neutrino/bin/build.js

```diff
@@ -7,10 +7,7 @@ module.exports = (middleware, options) => {
   return build(middleware, options)
     .fork((errors) => {
       spinner.fail('Building project failed');
-      errors.forEach((err) => {
-        console.error(err.stack || err);
-        err.details && console.error(err.details);
-      });
+      errors.forEach(err => console.error(err));
```

Comment:
Or if you prefer, put a `logError` or something in utils that has the same thing.

```js
const logError = unary(console.error);

module.exports = {
  // ...
  logError,
  // ...
};
```